app/apps/cases/models.py

```diff
@@ -1,12 +1,27 @@
 import uuid
 
 from apps.addresses.models import Address
-from apps.camunda.services import CamundaService
 from apps.events.models import CaseEvent, ModelEventEmitter
 from django.conf import settings
 from django.db import models
-from django.db.models.signals import post_init, post_save
-from django.dispatch import receiver
+from django.utils import timezone
+
+
+class CaseTeam(models.Model):
+    name = models.CharField(max_length=255, null=False)
```

Comment:
null=False is default mag weggelaten worden

Reply:
Done